### DISCUSSION TITLE
Connecting to a FTPES NAS

### DISCUSSION THREAD

**Discussion by Marcounet** [iterate-ch/cyberduck#16110] [Q&A]
Hello,
My ISP is offering to convert my router's hard drive into a NAS. The problem is that it can only be accessed using the FTPES protocol. I've tried FTP-SSL and SFTP without success. Is there a way to connect using Cyberduck/Mountain Duck?
If so, how do I go about it?

**Comment by tatze96** [comment]
Please use the [FTPS](https://docs.cyberduck.io/protocols/ftp/#tls-connections-ftps) connection profile to connect to your FTPES server. SFTP won't work, as it is a different connection protocol.

Do you get any error messages? If so, add a screenshot. Ensure to add the screenshot through the GitHub web interface, as it wouldn't be visible to us otherwise.

**Comment by Marcounet** [comment]
I'd tried but it don't work. Here two screenshot to help you.
<img width="547" alt="Capture d’écran 2024-07-13 à 09 57 46" src="https://github.com/user-attachments/assets/ac5b2251-ceb8-4c6b-93c2-b09cecf1ca80">
<img width="547" alt="Capture d’écran 2024-07-13 à 09 57 52" src="https://github.com/user-attachments/assets/8140f4fa-cefa-4294-a418-023cf7f45ac5">
Edit 1:
And I tried with these adresses and the same error :
ftpes://marcounet.freeboxos.fr
ftpes://marcounet.freeboxos.fr:56050
ftpes://freebox@marcounet.freeboxos.fr
ftpes://freebox@marcounet.freeboxos.fr:56050

Edit 2:
And I tried with these hosts:
<img width="547" alt="Capture d’écran 2024-07-13 à 10 11 44" src="https://github.com/user-attachments/assets/2b548b35-a504-4632-ba6b-31b6bc2eceaf">
<img width="547" alt="Capture d’écran 2024-07-13 à 10 11 50" src="https://github.com/user-attachments/assets/6b417f8b-9272-4788-ab6a-a1cce2ef40bf">

**Reply by tatze96** [reply]
Please select the FTPS connection profile instead of the SFTP connection profile at the top of the bookmark configuration window. Also, ensure that the hostname within the "Server" field is `marcounet.freeboxos.fr` as the remaining parts cause the DNS lookup error from the first screenshot.

You may have to re-enter the custom port and your credentials after adjusting the connection profile.

**Comment by Marcounet** [comment]
Once more for you:
When I try to my first connection with the server with FileZilla (but I prefer CyberDuck and MountainDuck) I receive a certificate like this:
<img width="768" alt="Capture d’écran 2024-07-13 à 10 45 37" src="https://github.com/user-attachments/assets/f429e3b0-cc09-40d2-8820-924fc3c8637f">

**Comment by Marcounet** [comment]
Hello,

I don’t try your solution yet, because my provider has some problems to send me my new device to connect me to the Internet. So be patient, like me, and I reply to you when I can test your solution.

Best regards, Marc Blériot.

---===***===---
Marc Blériot
– e-mail : ***@***.***
– mobile : (+33) 7 49 38 52 04

**Comment by Marcounet** [comment]
Hello Laura,

At last I've received my new box, so I can use my shared disk again.
As a reminder, the shared hard disk managed by my ISP (it's in the access box) only gives me a FTPES link to connect, which is only known by FileZilla.
You asked me to test an FTPS connection (unknown in the French version of Cyberduck, instead there's FTP-SSL), as you suggested in your last email, I checked my username, port and, finally, my password. And I've checked everything and it still doesn't work. I know that the first time I connect to my shared disk in the box with FileZilla, it asks me to validate a certificate.
What should I do to connect to this shared disk with your great software and its friend Mountainduck?

Translated with DeepL.com (free version)

Best regards, Marc Blériot.

---===***===---
Marc Blériot
– e-mail : ***@***.***
– mobile : (+33) 7 49 38 52 04
